projects/demo/src/modules/components/input-date-time/examples/4/index.ts

```diff
@@ -0,0 +1,23 @@
+import {Component} from '@angular/core';
+import {FormControl} from '@angular/forms';
+import {changeDetection} from '@demo/emulate/change-detection';
+import {encapsulation} from '@demo/emulate/encapsulation';
+import {TUI_DATE_TIME_VALUE_TRANSFORMER} from '@taiga-ui/kit';
+
+import {ExampleDateTimeTransformer} from './value-transformer';
+
+@Component({
+    selector: 'tui-input-date-time-example-4',
+    templateUrl: './index.html',
+    providers: [
+        {
+            provide: TUI_DATE_TIME_VALUE_TRANSFORMER,
+            useValue: new ExampleDateTimeTransformer(),
```

Comment:
`useClass` is a more appropriate way, please change it in other places too

Reply:
done